src/core/decoders/number.ts

```diff
@@ -22,16 +23,26 @@ export const number = (): FieldDecoder<number | ""> => {
     decode: value => {
       switch (typeof value) {
         case "number":
-          return Number.isFinite(value)
-            ? { ok: true, value }
-            : { ok: false, value };
+          return Number.isFinite(value) ? { ok: true, value } : { ok: false };
```

Comment:
minor, imho it's worth mentioning in docs, what values/types are accepted, as the area wider than finite numbers only
same for the rest of the decoders

Reply:
each decoders has jsdoc attached to it, check them out :)

for number there is:
```
/**
 * Define field of type `number | ""` (empty string is needed to represent state of empty number inputs)
 * Does not accept infinite numbers: `Infinity`, `-Infinity`, `NaN`.
 * Default initial value will be `""`.
 *
 * @example
 * ```
 * const Schema = createForm.schema(fields => ({
 *   x: fields.number() // x: number | ""
 * }));
 * ```
 */
```

Reply:
jsdoc improved